Azure/sonic-buildimage
# [docker-platform-monitor] Convert ledd from polling-based to subscription-based model

**- What I did**

  - Instead of polling State DB for link state changes, ledd now subscribes for changes to PORT_TABLE and reacts to notifications.
  - Also clean up Dockerfile.j2 template for docker-platform-monitor

**- How to verify it**

  - Flap all links and verify respective front panel LEDs turn on when link is up and off when link is down.
